idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -178,27 +162,18 @@ class CompressionStageData(PressureChangerData):
             "block data argument.",
         ),
     )
-    CONFIG.declare(
-      "first_stage",
-      ConfigValue(
-        default=True,
-        domain=In([True, False]),
-        description="Flag to define first stage",
-        doc="If this is the first stage, user must provide IGV value."
-        "Otherwise, user may provide the value of parameter A, B, C"))
-
-    ################################################
 
     def build(self):
         super().build()
 
+        # add ports to compressor
+        self.add_inlet_port(name="vapor_inlet", block=self.control_volume)
+        self.add_outlet_port(name="vapor_outlet", block=self.control_volume)
+
+        # Pyomo arc connect vapor inlet steam with control volume inlet
+        TransformationFactory("network.expand_arcs").apply_to(self)
```

Comment:
What are you trying to do here? As far as I can tell, this is build on a standard compressor model, which already has inlet and outlet ports.

Lines 170 and 171 seem to be just duplicating the existing Ports, and there is no need to expand Arcs, as there are no Arcs in the model as far as I can tell.